File: src/main/java/fi/finlit/edith/ui/services/svn/SubversionException.java
package fi.finlit.edith.ui.services.svn;

public class SubversionException extends RuntimeException {
    private static final long serialVersionUID = 2137588590021188211L;

    public SubversionException(Throwable t) {
        super(t);
    }

    public SubversionException(String msg, Throwable t) {
        super(msg, t);
    }
}

File: src/main/scala/fi/finlit/edith/ui/services/svn/SubversionException.scala
package fi.finlit.edith.ui.services.svn

//remove if not needed
import scala.collection.JavaConversions._

@SerialVersionUID(2137588590021188211L)
class SubversionException() extends RuntimeException {

  def this(t: Throwable) {
    super(t)
  }

  def this(msg: String, t: Throwable) {
    super(msg, t)
  }
}
